### ISSUE TITLE
Guess MIME type when serving static resources

### ISSUE THREAD

**Issue by tipsy** [perwendel/spark#545]
I propose:

```text
if file has extension
    mimeMap[extension]
else 
    probe
```

**Comment by tipsy** [comment]
Working on this in #551. I would appreciate any tips about guessing MIME-type when there's no extension (```Files.probeContentType()``` is useless)

**Comment by kshep92** [comment]
See my comment on #551 Allowing the user to register their custom mime types this way could save you from having to come up with a method guess mime types.

**Comment by tipsy** [comment]
Fixed in #589
